cloud-init/write-kali-aws-config-artifact-export.tpl.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# This is a Terraform template file, and the input variables are
+# passed in via templatefile().
+#
+# shellcheck disable=SC2154
```

Comment:
Do we need a reason here?

Reply:
See [here](https://github.com/cisagov/cool-assessment-terraform/pull/213#discussion_r1128306069).